src/GitParamTabExpansion.ps1

```diff
@@ -150,8 +150,24 @@ $gitParamValues = @{
     log = @{
         decorate = 'short full no'
         'no-walk' = 'sorted unsorted'
-        pretty = 'oneline short medium full fuller email raw'
-        format = 'oneline short medium full fuller email raw'
+        pretty = {
+            param($ref)
+            $completions = @()
+            gitConfigKeys $ref 'pretty' | ForEach-Object { $completions += $_ }
+            'oneline short medium full fuller email raw' -split ' ' |
```

Comment:
Would be nice to reduce some of this duplication. Two ideas:

1. Extract a `$defaultPrettyFormats` variable (with new options `reference` and `mboxrd`; [docs](https://git-scm.com/docs/pretty-formats)), probably pre-`split`
2. Update `gitConfigKeys` to accept an optional third parameter with additional values to include (i.e. `gitConfigKeys $format 'pretty' $defaultPrettyFormats`), so we can filter/sort all at once

And IMO the `$section` parameter looks better first, since that specifies which keys we're about to filter. We'd end up with:

```pwsh
pretty = {
    param($format)
    gitConfigKeys 'pretty' $format $defaultPrettyFormats
}
```